packages/kokkos-kernels/src/common/KokkosKernels_Utils.hpp

```diff
@@ -393,7 +392,7 @@ struct FillSymmetricLowerEdgesHashMap{
     in_lno_nnz_view_t adj_,
     hashmap_t hashmap_,
     out_lno_row_view_t pre_pps_,
-    bool lower_only_ = false
+    bool /* lower_only_ */ = false
```

Comment:
hmm .. what is this doing ?

Reply:
@srajama1 I just copied in the whole Utils.hpp since there were a couple of things the MTSGS needed from there (an array sum-reducer, and the updated team/vector size heuristics). This change removing an unused parameter warning:
```
commit d2032fdf9e1893a469285a184a52e018920e9ddc
Author: Jason M. Gates <jmgate@sandia.gov>
Date:   Tue Apr 23 16:54:50 2019 -0600

    Remove Unused Parameter Warnings
    
    Comment out parameter names in function definitions to avoid
    `-Wunused-parameter` warnings.  If their use is hidden behind an #ifdef,
    use an #else block to avoid the warning appropriately.
```

Reply:
Based on the name of the functor, I guess "lower only" is redundant since that's always what it does.

Reply:
I would say delete the line :)

Reply:
BTW, doesn't have to block this commit. It is just silly.